src/main/java/appeng/tile/misc/TileCharger.java

```diff
@@ -89,19 +88,19 @@ public boolean readFromStream_TileCharger( final ByteBuf data )
 		{
 			final IAEItemStack item = AEItemStack.loadItemStackFromPacket( data );
 			final ItemStack is = item.getItemStack();
-			this.inv.setInventorySlotContents( 0, is );
+			this.inv.setStackInSlot( 0, is );
 		}
 		catch( final Throwable t )
 		{
-			this.inv.setInventorySlotContents( 0, ItemStack.EMPTY );
+			this.inv.setStackInSlot( 0, ItemStack.EMPTY );
 		}
 		return false; // TESR doesn't need updates!
 	}
 
 	@TileEvent( TileEventType.NETWORK_WRITE )
 	public void writeToStream_TileCharger( final ByteBuf data ) throws IOException
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00100)